### PR TITLE
Add new aliases

### PR DESCRIPTION
**Pull request by CrestfallPS**

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -75,15 +77,19 @@ exports.BattleAliases = {
...
 	"megasalamence": "Salamence-Mega",
 	"megamence": "Salamence-Mega",
+	"megascept": "Sceptile-Mega",
 	"megasceptile": "Sceptile-Mega",
 	"megascizor": "Scizor-Mega",
 	"megasharpedo": "Sharpedo-Mega",
 	"megaslowbro": "Slowbro-Mega",
 	"megasteelix": "Steelix-Mega",
 	"megaswampert": "Swampert-Mega",
+	"megapert": "Swampert-Mega",
```
Comment (ascriptmaster) [inline data/aliases.js]: no megaswamp? smh

**Comment by Zarel** [comment]
Um... the alias list is getting sort of huge... are these all used?

**Comment by Zarel** [comment]
ehh, it's still reasonably small I guess

**Comment by kotarou3** [comment]
Maybe start logging and aggregating alias uses for statistics :p

**Pull request merged**